packages/devtools/client/styles/global.css

```diff
@@ -29,7 +29,8 @@ html.dark {
 .v-popper--theme-tooltip .v-popper__arrow-inner,
 .v-popper--theme-dropdown .v-popper__arrow-inner {
   visibility: visible;
-  --at-apply: border-white 'dark:border-hex-121212';
+  /* prettier-ignore */
+  --at-apply: border-white dark:border-hex-121212;
```

Comment:
```suggestion
  --at-apply: border-white dark-border-hex-121212;
```

Reply:
I applied your suggested change. It doesn't break eslint, and compile well.